STREAMING/SetPedPopulationBudget.md

```diff
@@ -5,7 +5,7 @@ ns: STREAMING
 
 ```c
 // 0x8C95333CFC3340F3 0xD2D026CD
-void SET_PED_POPULATION_BUDGET(int p0);
+void SET_PED_POPULATION_BUDGET(int p5);
```

Comment:
Hello,

Is this an error? Cause this is wrong.

Thanks

Reply:
This is the `budgetLevel` and it ranges from `0` to `4`.

Reply:
Isn't from `0` to `3`? Lowest is `0` for no peds and `3` is full budget.

Reply:
> Isn't from `0` to `3`? Lowest is `0` for no peds and `3` is full budget.

You're correct, my bad. Will edit my reply.